### DISCUSSION TITLE
Model or Problem?

### DISCUSSION THREAD

**Discussion by IlianPihlajamaa** [IlianPihlajamaa/ModeCouplingTheory.jl#7] [General]
Do we prefer to use MCTModel or MCTProblem, I don't really mind...

**Comment by thvoigtmann** [comment]
It just occured to me that `solve(problem)` is a nice API, so I'm happy with `MCTProblem` :-)

**Comment by IlianPihlajamaa** [comment]
Daniele brought up `Equation`, which may be more natural than `Problem`, and has the nice `solve(equation)`.
